src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -111,7 +114,7 @@ private unsafe bool InitCultureData()
                 }
                 // We found a locale name, so use it.
                 // In vista this should look like a sort name (de-DE_phoneb) or a specific culture (en-US) and be in the "pretty" form
-                this.sSpecificCulture = new String(pBuffer, 0, result - 1);
+                _sSpecificCulture = new String(pBuffer, 0, result - 1);
```

Comment:
Hmm, I thought the code formatter was changing type names to their C# keywords where applicable, e.g. String => string... I guess not.